samples/pojo-sample/pojo-consumer/src/main/resources/META-INF/spring/pojo.consumer.bean.xml

```diff
@@ -20,8 +20,10 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xmlns:cse="http://www.huawei.com/schema/paas/cse/rpc"
   xmlns:context="http://www.springframework.org/schema/context"
-  xsi:schemaLocation="http://www.springframework.org/schema/beans classpath:org/springframework/beans/factory/xml/spring-beans-3.0.xsd
-	http://www.huawei.com/schema/paas/cse/rpc classpath:META-INF/spring/spring-paas-cse-rpc.xsd">
+  xsi:schemaLocation="
+		http://www.springframework.org/schema/beans classpath:org/springframework/beans/factory/xml/spring-beans-3.0.xsd
+		http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd
+		http://www.huawei.com/schema/paas/cse/rpc classpath:META-INF/spring/spring-paas-cse-rpc.xsd">
```

Comment:
It doesn't make scene that we have name space of paas/cse/rpc here.

Reply:
Ok,wait a moment